src/main/java/com/iridium/iridiumskyblock/PlaceholderBuilder.java

```diff
@@ -0,0 +1,41 @@
+package com.iridium.iridiumskyblock;
+
+import com.iridium.iridiumskyblock.database.Island;
+import com.iridium.iridiumskyblock.database.IslandBlocks;
+import com.iridium.iridiumskyblock.database.User;
+import com.iridium.iridiumskyblock.utils.Placeholder;
+
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class PlaceholderBuilder {
+    private final List<Placeholder> placeholderList = new ArrayList<>();
+
+    public PlaceholderBuilder() {
+        placeholderList.add(new Placeholder("prefix", IridiumSkyblock.getInstance().getConfiguration().prefix));
```

Comment:
You should probably write a method to do this... Much duplicate code at the moment

Reply:
a method to do what? I am confusion

Reply:
To register a placeholder replacement

Reply:
No cuz if its an object we can parse it to ItemStacks and other sutff, anyways ima merge this. if u think its shit do a pr
:heart: